### ISSUE TITLE
3.0.4 breaks on ubuntu when using less-plugin-glob

### ISSUE THREAD

**Issue by bryce-larson** [plus3network/gulp-less#201]
Not sure what the root cause is but v3.0.4 breaks with the following error message.
This was working fine with v3.0.3. 

Might be an issue with one of the plugins less-plugin-glob or toolkit-less-css, but they haven't changed.
NOTE: this works fine on OSX with v3.0.4

error message from gulp, where PATH is obfuscated
```
Potentially unhandled rejection [2] Error in plugin 'gulp-less'
Message:
    './app-ui/templates/**/*.less' wasn't found. Tried - /home/ubuntu/<PATH>/front-end/app-ui/templates/**/*.less,app-ui/templates/**/*.less in file /home/ubuntu/.<PATH>/front-end/main.less line no. 147
Details:
    type: File
    filename: /home/ubuntu/<PATH>/front-end/main.less
    index: 5527
    line: 147
    callLine: NaN
    callExtract: undefined
    column: 0
    extract: @import "./app-ui/layouts/**/*";,@import "./app-ui/templates/**/*";,
    lineNumber: 147
    fileName: /home/ubuntu/front-end/main.less
```
gulpfile.coffee
```coffeescript
gulp = require("gulp")
less = require("gulp-less")
lessGlob = require("less-plugin-glob")
toolkit = require("toolkit-less-css")
 
gulp.task("styles:compile", ->
  gulp.src([
    "front-end/main.less"
  ]).pipe(less({
      strictMath: true,
      plugins: [toolkit, lessGlob]
    }))
)
```
line 146-7 from main.less
```less
@import "./app-ui/layouts/**/*";
@import "./app-ui/templates/**/*";
```

**Comment by zaerl** [comment]
Confirmed in 3.0.4 as well.

**Comment by craigjennings11** [comment]
Can you check and see if #203 fixes this issue?

**Comment by zaerl** [comment]
Yes, it solves the problem. Thanks @craigjennings11

**Comment by contra** [comment]
Fixed in 3.0.5

**Comment by bryce-larson** [comment]
Thanks @craigjennings11, it worked.
